### ISSUE TITLE
gamx update schema failing

### ISSUE THREAD

**Issue by daethnir** [taers232c/GAMADV-XTD3#12]
Gamx should be able to add new fields to schemas. I'm currently getting 'does not exist'. Verified that the same command on the same schema w/ same values works for jay's 'gam' command:

$ gamx version
GAM 4.86.03 - https://github.com/taers232c/GAMADV-XTD3

$ gamx update schema Entitlements     field foocloud indexed type bool endfield
Schema: Entitlements, Update Failed: Does not exist

$ gam version
GAM 4.40 - https://git.io/gam

$ gam update schema Entitlements     field foocloud indexed type bool endfield
Updated user schema Entitlements

**Comment by taers232c** [comment]
What does this show: gamx info schema Entitlements

Ross
--
ross.scroggs@gmail.com

**Comment by daethnir** [comment]
Schema: Entitlements
  displayName: Entitlements
  schemaId: 49AaXhuNRSaCyjBPz34xvw==
  Field: barfield
    fieldType: BOOL
    indexed: True
    displayName: barfield
    readAccessType: ADMINS_AND_SELF
    fieldId: KnGoFAg6RcyYPzDzYo5_ZQ==
  Field: bazfield
    fieldType: BOOL
    displayName: bazfield
    readAccessType: ADMINS_AND_SELF
    fieldId: Y0GYwpnySoGCrcDyDlDdkA==

**Comment by daethnir** [comment]
Note: I made a new 'deleteme' schema yesterday for testing as well. It
exhibited the same problem as I saw with Entitlements, where gamx could not
add fields but gam could.

Today, however, gamx *could* add fields, using the exact same command from
my history.

Not sure what's going on here. Do not want to add to our actual
Entitlements schema for testing purposes though, because it's in production
use.
